app/src/main/java/com/hedvig/app/ui/compose/composables/screens/GenericErrorScreen.kt

```diff
@@ -0,0 +1,64 @@
+package com.hedvig.app.ui.compose.composables.screens
+
+import android.content.Context
+import android.content.res.Configuration
+import android.util.AttributeSet
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.AbstractComposeView
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.hedvig.app.R
+import com.hedvig.app.ui.compose.theme.HedvigTheme
+
+class GenericErrorScreenView @JvmOverloads constructor(
+  context: Context,
+  attrs: AttributeSet? = null,
+  defStyle: Int = 0
+) : AbstractComposeView(context, attrs, defStyle) {
```

Comment:
ktlint is probably gonna cry about this missing trailing comma after `defStyle: Int = 0`  :D